hod/config/template.py

```
@@ -105,18 +105,17 @@ def register_templates(template_registry, config_opts):
     for ct in templates:
         template_registry.register(ct)
 
-def mklocalworkdir(workdir, label=None):
+def mklocalworkdir(workdir):
     '''
     Construct the pathname for a workdir with a path local to this
     host/job/user.
     '''
     user = _current_user()
     pid = os.getpid()
+    jobid = os.getenv('PBS_JOBID', '')
```

yes please, check whether `jobid` is `None`, and raise an error if it is to complain that `$PBS_JOBID` is not defined